src/coverlet.core/CoverageResult.cs

```diff
@@ -149,22 +149,23 @@ public ThresholdTypeFlags GetThresholdTypesBelowThreshold(CoverageSummary summar
                         double line = summary.CalculateLineCoverage(Modules).AverageModulePercent;
                         double branch = summary.CalculateBranchCoverage(Modules).AverageModulePercent;
                         double method = summary.CalculateMethodCoverage(Modules).AverageModulePercent;
+                        var numModules = Modules.Count;
```

Comment:
We already got the avg we don't need to divide by num of modules. We're changing old behavior.

Reply:
removed numModules. can you please check?

Reply:
@MarcoRossignoli, can I resolve this conversation? or is there something else that must be fixed?